app/models/concerns/reviews/validations.rb

```diff
@@ -64,14 +65,27 @@ def validate_extra_attributes?
       SHOW_REVIEW_EXTRA_ATTRIBUTES
     end
 
-    def validate_required_tag
+    def validate_required_business_unit_tag
       business_unit_type = plan_item&.business_unit&.business_unit_type
       is_invalid = business_unit_type&.require_tag &&
         taggings.reject(&:marked_for_destruction?).blank?
 
       errors.add :taggings, :blank if is_invalid
     end
 
+    def validate_required_tags
+      if will_save_change_to_scope?
+        required_tags = REVIEW_SCOPES[scope]&.fetch(:require_tags, nil) || []
+        needed_tags   = required_tags.flat_map { |option| Tag.list.with_option option }.uniq
```

Comment:
```suggestion
        tag_options    = REVIEW_SCOPES[scope]&.fetch(:require_tags, nil) || []
        required_tags = tag_options.flat_map { |option| Tag.list.with_option option }.uniq
```

Y sacamos el needed que no me gusta =)